### ISSUE TITLE
Offizielle Doku

### ISSUE THREAD

**Issue by FeyP** [bundesAPI/luftqualitaet-api#1]
Die offizielle API-Doku findet ihr hier: https://www.umweltbundesamt.de/daten/luft/luftdaten/doc
OpenAPI-Spec-Datei: https://www.umweltbundesamt.de/sites/all/modules/custom/w21_air_data/doc/uba-air-data-api_public.yaml

**Comment by pixeldesu** [comment]
@FeyP Oh, das ist echt cool...ist eigentlich genau alles was ich selbst dokumentiert hab! 

Ist das irgendwo auf der Seite selbst verlinkt gewesen? Hab in den letzten 2 Wochen da viel Zeit drauf verbracht aber keinen Link zu dieser Doku gesehen.

**Comment by FeyP** [comment]
Ja, das tat mir auch ein bisschen Leid, dass du das aufwändig reverse engineered hast, obwohl es eigentlich auch eine Doku vom UBA dazu gibt. Ob die irgendwo verlinkt ist, weiß ich nicht, ich habe einfach mal gegoogelt nach "umweltbundesamt luftdaten api" und das kam dann bei mir als erstes Ergebnis. Es scheint auch schon einige andere Leute zu geben, die die API verwenden (ausweislich der Suchergebnisse), insofern muss es eigentlich schon irgendwo auch auffindbar dokumentiert sein, vermute ich.

Man könnte ja mal beim UBA anregen, die Doku auch dort auf der Seite zu verlinken, damit sie besser auffindbar ist. Ich habe auch die beiden Versionen jetzt nicht abgeglichen. Vielleicht kann man dem UBA ja auch einen Patch mit Ergänzungen der Dokumentation schicken, falls deine Doku besser ist.
